readme.md

```diff
@@ -89,6 +89,15 @@ Click the type names for complete docs.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
 
+### Template literal types
+
+*Note:* These require [TypeScript 4.1 or newer](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#template-literal-types).
+
+- [`CamelCase`](source/camel-case.d.ts) – Convert a typical casing to camel-case (`fooBar`).
```

Comment:
I think the description here could be clearer. Not clear what "typical casing" means.